asteroids_code/BoilerPlate/player.cpp

```diff
@@ -58,24 +64,51 @@ void Player::Update(const int w_width, const int w_height)
 
 void Player::MoveForward()
 {
-	//origin += vector2(move_speed * cos(angle_degree / 57.3), move_speed * sin(angle_degree / 57.3));
-	origin += Vector2(0.0, move_speed);
+	float angle_radians = conv.DegToRad(angle_degree);
+	origin += Vector2(-move_speed * sin(angle_radians), move_speed * cos(angle_radians));
+	//origin += Vector2(0.0, move_speed);
+	thruster_on = true;
 }
 
 void Player::RotateLeft()
 {
-	/*glRotatef(rotation_angle, 0.0, 0.0, 1.0);
-	angle_degree += 4;*/
+	angle_degree += rotation_angle;
+	if (angle_degree >= 360)
```

Comment:
You should have a utility function in your math utils class to handle this. Regardless, I don't think this is necessary.